### PR TITLE
Chore/file reorg

### PR DESCRIPTION
**Pull request by rhyslbw**
This PR renames the PortManager to PortMapper, and reorganises files in preparation for implementing types and components common to the execution engine and bundle server.
See commit log

### CODE REVIEW

File: src/lib/expressPromises.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import express from 'express'
+import http from 'http'
+
+export const expressPromises = {
```
Comment (Sam-Jeston) [inline src/lib/expressPromises.ts]: I think this could be named better

Maybe `expressEventPromiseHandler`

File: src/lib/httpPromises.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import * as http from 'http'
+
+export const httpPromises = {
```
Comment (Sam-Jeston) [inline src/lib/httpPromises.ts]: Same here

**Review by Sam-Jeston — CHANGES_REQUESTED** [review]
Just minor naming thoughts.

Also please remove this from the git cache and update .gitignore: src/execution_engine/routes.ts

**Comment by rhyslbw** [comment]
The change to gitignore was committed in https://github.com/input-output-hk/smart-contract-backend/pull/72/commits/729ce0bc2104037bc459c3318f1e63e6f2c6628e

**Edit:** Sorry, didn't see that was committed already. It's been removed from the cache

**Review by Sam-Jeston — APPROVED** [review]
💯

**Pull request merged**